sandy/gls.py

```diff
@@ -22,7 +21,7 @@
         ]
 
 x_prior = [1, 2, 3]
-y_extra = pd.Series([2, 3, 4], index=[1, 2, 3])
+y_extra = np.array([2, 3, 4])
```

Comment:
@EnricaBelfiore, what are all these things? If not needed we can remove them

Reply:
They have been inserted and used in the docstring example of `sandy.gls._y_calc()`. They will be removed (and the function as well) in another PR